src/main/java/mate/academy/springboot/datajpa/model/Product.java

```diff
@@ -0,0 +1,25 @@
+package mate.academy.springboot.datajpa.model;
+
+import java.math.BigDecimal;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@Entity
+@Table(name = "products")
+public class Product {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String title;
+    private BigDecimal price;
+    @OneToOne
```

Comment:
Why one to one? Here should be many to many.

Reply:
Logically yes, but in task: category (one product can have one category)?